pkg/minikube/bootstrapper/kubeadm/templates.go

```diff
@@ -44,6 +44,45 @@ nodeName: {{.NodeName}}
   {{$i}}: {{$val}}{{end}}
 {{end}}`))
 
+var kubeadmConfigTemplateV1Alpha3 = template.Must(template.New("kubeadmConfigTemplate-v1alpha3").Funcs(template.FuncMap{
+	"printMapInOrder": printMapInOrder,
+}).Parse(`apiEndpoint:
+  advertiseAddress: {{.AdvertiseAddress}}
+  bindPort: {{.APIServerPort}}
+apiVersion: kubeadm.k8s.io/v1alpha3
+bootstrapTokens:
+- groups:
+  - system:bootstrappers:kubeadm:default-node-token
+  token: 599s02.8i3w2yyyay8t0onm
```

Comment:
How was this value arrived at?

Reply:
I have used `kubeadm config upgrade`, cleared all the default values, just by accident left this one. I will remove it.